scripts/setup_ollama.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "Starting Ollama server..."
+ollama serve &
+SERVER_PID=$!
+
+echo "Waiting for Ollama server to be active..."
+while [ "$(ollama list | grep 'NAME')" == "" ]; do
+  sleep 1
+done
+
+# Pull default models
+echo "Pulling embedded models..."
+ollama pull nomic-embed-text
+echo "Pulling default LLM models..."
+ollama pull phi3
```

Comment:
as we mount  `/root/.ollama` - i guess these are no-ops if we start the container again?

Reply:
how about using curl API, then you can run it this script the setup container?

https://github.com/ollama/ollama/blob/main/docs/api.md#pull-a-model

Reply:
pass host and LLM model as parameter (see script call)